lte/gateway/Vagrantfile

```diff
@@ -43,7 +43,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # 2. Any changes to the magma networking should be propagated to magma_deb
     magma.vm.network "private_network", ip: "192.168.60.142", nic_type: "82540EM"
```

Comment:
You probably want to use internal for the test <-> dev vm connection as well

Reply:
Done